docs/docs/APIGuide/Layers/Convolution-Layers.md

```diff
@@ -802,6 +802,124 @@ output m is : [[[-0.04712385  0.21949144  0.0843184   0.14336972]
   [-0.38188276 -0.36746511 -0.37627563 -0.34141305]]]
 ```
 
+## SpatialSeperableConvolution ##
+
+**Scala:**
+```scala
+val m  = SpatialSeperableConvolution(nInputChannel, nOutputChannel, depthMultiplier, kW, kH, sW, sH, pW, pH, hasBias, dataFormat, wRegularizer = null,bRegularizer = null, pRegularizer = null, initDepthWeight = null, initPointWeight = null, initBias = null)
```

Comment:
Separ...

Reply:
Any default value for pW, pH, hasBias, dataFormat?

Reply:
let me make it consist with python

Reply:
but such code written in Python is not well formatted....